src/bank.rs

```diff
@@ -1301,39 +1335,55 @@ mod tests {
         mint: &Mint,
         keypairs: &[Keypair],
     ) -> impl Iterator<Item = Entry> {
+        let mut last_id = mint.last_id();
         let mut hash = mint.last_id();
         let mut entries: Vec<Entry> = vec![];
+        let mut num_hashes = 0;
         for k in keypairs {
             let txs = vec![Transaction::system_new(
                 &mint.keypair(),
                 k.pubkey(),
                 1,
-                hash,
+                last_id,
             )];
             let mut e = ledger::next_entries(&hash, 0, txs);
             entries.append(&mut e);
             hash = entries.last().unwrap().id;
+            let tick = Entry::new_mut(&mut hash, &mut num_hashes, vec![]);
+            last_id = hash;
```

Comment:
This is confusing. Why two variables `last_id` and `hash`?

Reply:
because not all hashes can be last_ids